server/migrations/postgres/20240422191813_create-files.up.sql

```diff
@@ -0,0 +1,8 @@
+create table files (
+    id uuid primary key,
+    name text not null,
```

Comment:
Big question, I don't remember our previous discussion: should this be case sensitive or not?

Reply:
Right, I completely forgot about it.

Lower case for the comparison but everything is saved as provided by the user.

```
this/is/my/directory/tex.txt
// equal to
This/IS/mY/directory/tExT.txt
```

but in the UI it should be displayed as `This/IS/mY/directory/tExT.txt`.

It can be misleading for the user but it solves several problems with mac os and windows.

Reply:
Ok, then I would use `citext` as the column type, because

> Essentially, it internally calls lower when comparing values. Otherwise, it behaves almost exactly like text.

Cit https://www.postgresql.org/docs/current/citext.html